### ISSUE TITLE
Can't get veyon 4.7.3 to run on Ubuntu 22.04

### ISSUE THREAD

**Issue by ridayc** [veyon/veyon#810]
I recently installed veyon from the ppa repository on a fresh install of Ubuntu 22.04. The installation works fine, however running veyon-configurator results in an immediate segmentation fault (core dumped). Running on Wayland I receive I get a warning: 
`Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.`

But even on KDE Plasma veyon-configurator crashes immediately.

I tried to build veyon from source however there seem to be issues with the cmake files finding the correct libvncclient libraries (even though libvncclient1 is at version 0.9.13 on Ubuntu).
Cmake issues the following warning:
```
-- Could NOT find LibVNCClient (missing: LibVNCClient_INCLUDE_DIRS LibVNCClient_LIBRARIES) (Required is at least version "0.9.13")
CMake Warning at core/CMakeLists.txt:43 (message):
  Performing internal build of LibVNCClient which requires additional
  development packages


CMake Warning at plugins/vncserver/x11vnc-builtin/CMakeLists.txt:10 (message):
  No suitable LibVNCClient library found therefore performing internal build
  of LibVNCServer as well to avoid binary compatibility issues


-- Could NOT find LibVNCServer (missing: LibVNCServer_INCLUDE_DIRS LibVNCServer_LIBRARIES) (Required is at least version "0.9.8")
```

Because of this it seems that an external version libvncserver is built. This however fails to build with various 
``` -Werror=deprecated-declarations``` errors being thrown on building 
```
Building C object core/CMakeFiles/veyon-core.dir/__/3rdparty/libvncserver/common/crypto_openssl.c.o
```

I had managed to have veyon 4.6 running on Ubuntu 21.04 and really appreciate veyon as a monitoring software.

It would be great if there are some fairly simply solutions to resolving these issues.

**Comment by tobydox** [comment]
Indeed Veyon is not compatible with Wayland yet, at least concerning the server part. However the latest 4.7 branch should build again on Ubuntu 22.04 (fix via 0c2e208f9badf88eb3d999ccd7f8cf4231416504).

**Comment by ridayc** [comment]
> Indeed Veyon is not compatible with Wayland yet, at least concerning the server part. However the latest 4.7 branch should build again on Ubuntu 22.04 (fix via [0c2e208](https://github.com/veyon/veyon/commit/0c2e208f9badf88eb3d999ccd7f8cf4231416504)).

Thanks for the quick fix of the cmake files!

The cmake changes allow Veyon to be compiled from source from branch 4.7 on.

I did a bit more testing. The configurator still crashes with an immediate segmentation fault. 
I can use a previous configuration file to start veyon-master. (Also using veyon-master from the ppa)
Veyon-Master opens stably (it crashed on KDE Plasma for some reason) using an old configuration file. It works the same for Wayland and X11 sessions. The only thing that doesn't work are preview windows (I'm connected via LDAP and built-in vnc).

Is there a way to get verbose from veyon-configurator output to check what might be causing the core dump segmentation faults?

**Comment by Kevin8675** [comment]
@ridayc Were you able to get Veyon client running on Ubuntu 22.04? If so, how? Thanks.

**Comment by dldnet** [comment]
Exactly the same problem. Ubuntu 22.04 too

**Comment by ridayc** [comment]
> @ridayc Were you able to get Veyon client running on Ubuntu 22.04? If so, how? Thanks.

The clients I steer are running on Windows (active directory), and I haven't tried running any Ubuntu clients. I was able to edit my configurations files using Veyon 4.5 (that one should still exist in the standard Ubuntu repositories). I was able to start Veyon Master on version 4.7 on Ubuntu 22.04. Everything works except, that I can't see the computer screen previews in the rooms (I can remote control clients though), and I can't project my screen to clients (but I can project one of the client's screens to all other clients).

I'm suspecting there's some type of problem with Wayland and with Qt as well (I can't get Veyon to work with Xorg and even master doesn't work on X11 on Kubuntu). I had managed to have Veyon 4.6 working on Ubuntu 21.10 (with screen previews), so some change from 4.6 to 4.7 (or from Ubuntu 21.10 to 22.04) must have caused the application to break.

**Comment by brunovila** [comment]
I get a Sementation fault error after installed in Ubuntu 22.04, (wayland disabled) when I run
`sudo veyon-configurator`

**Comment by Kevin8675** [comment]
Is there any update on a fix for this?

**Comment by tobydox** [comment]
Issue will be fixed in upcoming 4.7.4 release.
